packages/resolve-es-file/src/index.js

```diff
@@ -4,16 +4,17 @@ const ENCODING = 'utf8';
 
 const loadEvents = path =>
     new Promise((resolve, reject) =>
-        fs.readFile(
-            path,
-            ENCODING,
-            (err, content) =>
-                (err && err.code !== 'ENOENT'
-                    ? reject(err)
-                    : resolve(content ? JSON.parse(`[${content.replace(/,$/, '')}]`) : []))
-        )
+        fs.readFile(path, ENCODING, (err, content) => {
+            if (err && err.code !== 'ENOENT') {
+                reject(err);
+                return;
+            }
+            resolve(content ? JSON.parse(`[${content.replace(/,$/, '')}]`) : []);
+        })
     );
 
+const compareEvents = (a, b) => a.timestamp - b.timestamp;
```

Comment:
The timestamp is assigned outside the driver. So it can be a case when two sequentially saved events have such timestamps that timestamp of the first event is greater than that of the second.

Of course now there are no such examples.

So should it be removed now?